maptastic/build.boot

```diff
@@ -0,0 +1,39 @@
+(def +lib-release+ "0")
+(def +lib-version+ (str "0." +lib-release+ ".0"))
+
+(set-env!
+  :resource-paths #{"resources"}
+  :dependencies '[[cljsjs/boot-cljsjs "0.5.2"]])
+
+(require '[cljsjs.boot-cljsjs.packaging :refer :all]
+         '[boot.task.built-in :refer :all])
+
+(task-options!
+ pom  {:project     'cljsjs/maptastic
+       :version     +lib-version+
+       :description "Javascript/CSS projection mapping utility."
+       :url         "https://github.com/glowbox/maptasticjs"
+       :license     {"Copyright (C) 2015 by Glowbox LLC"
+                     "https://raw.githubusercontent.com/glowbox/maptasticjs/master/license.txt"}
+       :scm         {:url "https://github.com/cljsjs/packages"}})
+
+(deftask package []
+  (comp
+   (download :url
+             "https://raw.githubusercontent.com/glowbox/maptasticjs/master/build/maptastic.min.js"
```

Comment:
Like in this package: https://github.com/cljsjs/packages/blob/a486e6af0389a1d2754e3a21e7772f082b208815/emojione-picker/build.boot#L12